script.js

```diff
@@ -39,4 +36,175 @@ window.onresize = () => {
   }
 };
 
+// Model
+const overlayModal = document.getElementById('overlayModal');
+function openModal(modal) {
+  if (modal == null) return;
+  modal.classList.add('active');
+  window.scrollTo(0, 0);
+  overlayModal.classList.add('active');
+  navLinks.forEach((link) => {
+    link.style.color = '#c1c7d0';
+  });
+}
+
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
+  modal.innerHTML = '';
+
+  overlayModal.classList.remove('active');
+  navLinks.forEach((link) => {
+    if (w > 768) {
+      link.style.color = '#344563';
+    } else {
+      link.style.color = '#fff';
+    }
```

Comment:
- [ ] On mobile screens please make sure there's equal padding on left-right and content is in the center of the model to follow the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/217813419-f29423e0-a45a-423c-822a-8467b932e228.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/217813132-e8143512-04bc-44a3-aa6d-d69d2513966b.png)

Reply:
Fixed.